src/tests/unit/tests/electron/flux/store/electron-window-state-store.test.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { WindowStateActions } from '../../../../../../electron/flux/action/window-state-actions';
+import { ElectronWindowStateStore } from '../../../../../../electron/flux/store/electron-window-state-store';
+import { ElectronWindowStateStoreData } from '../../../../../../electron/flux/types/electron-window-state-store-data';
+import { createStoreWithNullParams, StoreTester } from '../../../../common/store-tester';
+
+describe('ElectronWindowStateStore', () => {
+    describe('constructor', () => {
+        it('has no side effects', () => {
+            const testSubject = createStoreWithNullParams(ElectronWindowStateStore);
+            expect(testSubject).toBeDefined();
+        });
+    });
+
+    it('returns device connect as the default state', () => {
+        const testSubject = new ElectronWindowStateStore(new WindowStateActions());
+        testSubject.initialize();
+
+        expect(testSubject.getState()).toEqual(testSubject.getDefaultState());
```

Comment:
can the default state be built using `createStoreWithNullParams(ElectronWindowStateStore).getDefaultState();` ??? that way you don't need to call `.initialize`

Reply:
we also wanted to validate the initialstate. it is easier to validate the initial state without using store tester & just call initialize.

Reply:
not sure what you're trying to test... but I can say this: the `BaseStoreImpl` class contains this function:
```typescript
    public initialize(initialState?: TState): void {
        this.state = initialState || this.getDefaultState();

        this.addActionListeners();
    }
```
and that's being tested here: `src\tests\unit\tests\background\stores\base-store.test.ts`
If you're trying to test the the initial state is being set by using `getDefaultState` that's already being cover elsewhere and there is no need to test that here too